### ISSUE TITLE
[OS X] Rendering bugs with variable-width fonts

### ISSUE THREAD

**Issue by angelXwind** [hexchat/hexchat#1260]
While it is generally frowned upon to use variable-width fonts in IRC, using variable-width fonts works just fine in Windows and Linux.

The most noticeable areas are places where HexChat creates clickable links - when you mouse over them, the underline will be misaligned, and the actual link text will shift to the right some amount of spaces.

The issue is reproducible by trying to use any variable-width font in HexChat under OS X.

**Comment by angelXwind** [comment]
Upon further inspection, it appears that this may be related to just how HexChat renders links - very minor rendering issues can be seen even when using fixed-width fonts too (like Menlo Regular 13pt).

**Comment by HenryNugraha** [comment]
I don't think this only occur on OS X. I'm on Linux Mint and I got this issue too. I also suspect the way HexChat renders links is the culprit. Selecting the text would fix it, though it will happen again when hovering links.

**Comment by TingPing** [comment]
@HenryNugraha I would agree HexChat's usage is at fault in some way, but I've never encountered these issues on any other platform nor has that ever been reported. Perhaps you can record a video of the issue?

**Comment by TingPing** [comment]
@HenryNugraha Also what version? A lot of rendering things have changed in 2.10.x.

**Comment by HenryNugraha** [comment]
@TingPing I'm using version 2.11.0.
![preview](https://cloud.githubusercontent.com/assets/9031009/5884114/36efcf96-a394-11e4-83c3-9b6b527722fe.gif)
Hope this helps describing the issue.

**Comment by TingPing** [comment]
@HenryNugraha Interesting, lets hope @RichardHitt can test it out and reproduce it.

**Comment by TingPing** [comment]
@HenryNugraha Can you pastebin your hexchat.conf?

**Comment by HenryNugraha** [comment]
@TingPing Sure thing, here it is. http://pastebin.com/4Q2yhVed

Also, after some testing I'm sure my font is causing this. I'm using LKLUG (size does not matter), and the texts are shifted to the right. It occurs on Lohit fonts too, except it shifts to the left (just by small amount, though). While this issue only happen when hovering and unhovering links, it won't shift if the links are the first text on the line.

I've tried reproducing this on Windows 7, and it also occurs with same symptoms mentioned above.

It seems changing the font will solve this problem.

**Comment by soofuzz** [comment]
yea even text is really blurry on Hex chat using OS X. 
![screen shot 2015-04-09 at 7 10 53 pm](https://cloud.githubusercontent.com/assets/11738103/7071168/3c94794a-deec-11e4-942b-404b19b346bf.png)

**Comment by TingPing** [comment]
@soofuzz Your issue is #828.

**Comment by Ombridride** [comment]
+1 Same issue on Mavericks, but with less difference between the mouse and the selected text (maybe 2/3/4 caracters)

**Comment by James-TR** [comment]
Woo, Yosemite too!

**Comment by Enrico204** [comment]
One year later, I have the same issue too
